### PR TITLE
Added unit test for pem.js

### PR DESCRIPTION
**Pull request by lovemaths**
Added unit tests for getPrivateKey, getCertificate and certToPem functions.

### CODE REVIEW

**Comment by polita** [comment]
It looks like the builds are failing. If you need help figuring out how the travis build works, ping @brandwe.

**Comment by lovemaths** [comment]
@polita Figured out the problem, it turns out that fs.readFileSync requires absolute path in travis build. Now it works.

File: test/pem_test.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+ */
+
+var extractAndCompare = function(extractFunc, pemFileName, correctValue) {
+  var pem = null;
+  var pemKeyOrCert = null;
+
+  try {
+    pem = fs.readFileSync(__dirname + '/resource/' + pemFileName, 'utf8');
+    pemKeyOrCert = extractFunc(pem);
+  } catch (e) {
```
Comment (polita) [inline test/pem_test.js]: What should we be doing here if there's an exception? Swallowing it silently probably isn't the right thing.

File: test/resource/correctKeyOrCert.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+exports.privateKey =
```
Comment (lovemaths) [inline test/resource/correctKeyOrCert.js]: I generated them using openssl with some fake values, so they have nothing sensitive.

File: test/pem_test.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+  'test certToPem': (test) => {
+    test.expect(1);
+    // tests here
+
+    var pemContent = pem.certToPEM('myCertificate');
+
+    var pass = ('myCertificate' === pem.getCertificate(pemContent));
+
+    test.ok(pass, 'should get the same certificate');
+
```
Comment (lovemaths) [inline test/pem_test.js]: removed.

File: test/pem_test.js (hunk 1 above)
Comment (lovemaths) [inline test/pem_test.js]: I just rewrote the test to print the error message in case of exception.

**Comment by polita** [comment]
:shipit:

**Pull request merged**